src/docs/development/platform-integration/web.md

```diff
@@ -9,32 +9,36 @@ As of the 1.12 release, web support for Flutter
 is available on the beta channel.
 **We don't recommend deploying a web app to production.**
 
+### What sort of apps are not ideal for Flutter on the web?
+
+While **we don’t recommend using Flutter to build static content
```

Comment:
This could go after we list the scenarios that it is good for within the same question